tests/test_translation_level_11.py

```diff
@@ -0,0 +1,44 @@
+import hedy
+from test_level_01 import HedyTester
+import hedy_translation
+from test_translating import check_local_lang_bool
+import textwrap
+
+
+# tests should be ordered as follows:
+# * Translation from English to Dutch
+# * Translation from Dutch to English
+# * Translation to several languages
+# * Error handling
+
+
+class TestsTranslationLevel11(HedyTester):
+    level = 11
+    keywords_from = hedy_translation.keywords_to_dict('en')
+    keywords_to = hedy_translation.keywords_to_dict('nl')
+
+    @check_local_lang_bool
+    def test_for_in_english_dutch(self):
+        code = textwrap.dedent("for counter in range 1 to 5\n"
+                               "    print counter")
+
+        result = hedy_translation.translate_keywords(code, from_lang="en", to_lang="nl", level=self.level)
+        expected = textwrap.dedent("voor counter in bereik 1 tot 5\n"
+                                   "    print counter")
+
+        self.assertEqual(result, expected)
+
+    @check_local_lang_bool
+    def test_for_in_dutch_english(self):
```

Comment:
Almost right :) 

with textwrap we don't need the \n's (that was sort of the point haha), so we can write it like this:

```suggestion
    def test_for_in_dutch_english(self):
        code = textwrap.dedent("""\  # this is to escape the newline that follows
        nummer is vraag 'hoe oud ben je'"
        voor counter in bereik 1 tot 5"
            voor count in bereik nummer tot 0"
                print 'hoi' counter""")
```